File: src/main/Java/boj_step3/boj8393.java
import java.io.*;

public class boj8393 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        int sum = n;
        while (n-- > 0) { sum += n; }
        writer.append(String.valueOf(sum));
        writer.close();
    }
}

File: src/main/Kotlin/others/boj8393.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    var sum = 0
    for (i in 1..n) sum += i
    w.write(sum.toString())
    w.close()
}